src/docs/guide/introduction.md

```diff
@@ -5,7 +5,7 @@ outline: deep
 
 <script setup>
 import { VPTeamMembers } from 'vitepress/theme'
-import { COMMUNITY_TEAM_MEMBERS, CORE_TEAM_MEMBERS } from '../../../.vitepress/theme/constants/team'
+import { COMMUNITY_TEAM_MEMBERS, CORE_TEAM_MEMBERS } from '@constants/team'
```

Comment:
It's cumbersome to modify Relative paths for translated version of documentation, which are stored in `/src/<locale>/*`.

For convenience, I defined Vite aliases so that we can just copy the original file to start translation. (see `config.mts` for more detail).